cherrypy/test/sessiondemo.py

```diff
@@ -2,7 +2,7 @@
 """A session demonstration app."""
 
 import calendar
-from datetime import datetime
+from datetime import datetime, UTC
```

Comment:
Looks like `UTC` needs to be accessed differently:
`E   ImportError: cannot import name 'UTC'`

Reply:
At least under Python 3.6...

Reply:
```suggestion
from datetime import datetime

try:
    from datetime import UTC  # Python 3.11+
except ImportError:
    # Python 3.6-3.10
    from datetime import timezone as _timezone
    UTC = _timezone.utc
    del _timezone
```